### PR TITLE
Better error messages

### PR DESCRIPTION
**Pull request by duckofyork**

### CODE REVIEW

**Comment by duckofyork** [comment]
!rtg

File: grakn-core/src/main/java/ai/grakn/exception/GraqlQueryException.java [hunk 1]
```diff
@@ -146,7 +150,7 @@ public static GraqlQueryException insertMetaType(Label label, SchemaConcept sche
 
     /**
      * Thrown when a concept is inserted with multiple properties when it can only have one.
-     *
+     * <p>
```
Comment (fppt) [inline grakn-core/src/main/java/ai/grakn/exception/GraqlQueryException.java]: Have a 🐐  to mark this incorrect use of a `<p>` tag. I.e. it has no closing tag.

File: grakn-core/src/main/java/ai/grakn/exception/GraqlQueryException.java [hunk 2]
```diff
@@ -155,7 +159,7 @@ public static GraqlQueryException insertMultipleProperties(String property, Obje
 
     /**
      * Thrown when a property is inserted on a concept that already exists and that property can't be overridden.
-     *
+     * <p>
      * For example: {@code match $x isa name; insert $x val "Bob";}
      */
     public static GraqlQueryException insertPropertyOnExistingConcept(String property, Object value, Concept concept) {
```
Comment (fppt) [inline grakn-core/src/main/java/ai/grakn/exception/GraqlQueryException.java]: 🐐

File: grakn-core/src/main/java/ai/grakn/exception/GraqlQueryException.java [hunk 3]
```diff
@@ -164,7 +168,7 @@ public static GraqlQueryException insertPropertyOnExistingConcept(String propert
 
     /**
      * Thrown when a property is inserted on a concept that doesn't support that property.
-     *
+     * <p>
```
Comment (fppt) [inline grakn-core/src/main/java/ai/grakn/exception/GraqlQueryException.java]: 🐐

File: grakn-core/src/main/java/ai/grakn/exception/GraqlQueryException.java [hunk 4]
```diff
@@ -173,7 +177,7 @@ public static GraqlQueryException insertUnexpectedProperty(String property, Obje
 
     /**
      * Thrown when a concept does not have all expected properties required to insert it.
-     *
+     * <p>
```
Comment (fppt) [inline grakn-core/src/main/java/ai/grakn/exception/GraqlQueryException.java]: 🐐

File: grakn-core/src/main/java/ai/grakn/exception/GraqlQueryException.java [hunk 5]
```diff
@@ -182,7 +186,7 @@ public static GraqlQueryException insertNoExpectedProperty(String property, VarP
 
     /**
      * Thrown when attempting to insert a concept that already exists.
-     *
+     * <p>
```
Comment (fppt) [inline grakn-core/src/main/java/ai/grakn/exception/GraqlQueryException.java]: 🐐

File: grakn-test/test-integration/src/test/java/ai/grakn/test/graql/analytics/AnalyticsTest.java [hunk 6]
```diff
@@ -127,6 +133,24 @@ public void testNullResourceDoesntBreakAnalytics() throws InvalidKBException {
     }
 
     @Test
+    public void testSubgraphHasRule() {
```
Comment (fppt) [inline grakn-test/test-integration/src/test/java/ai/grakn/test/graql/analytics/AnalyticsTest.java]: sad test name :(

**Comment by duckofyork** [comment]
@fppt all the goats have been murderd :D

**Comment by fppt** [comment]
Goat murders approved

**Comment by duckofyork** [comment]
!rtg

**Pull request merged**